src/tests/ftest/rebuild/delete_objects.py

```diff
@@ -90,7 +86,9 @@ def test_rebuild_delete_objects(self):
         Use Cases:
             foo
 
-        :avocado: tags=all,large,full_regression,rebuild,rebuilddeleteobject
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm,large,rebuild
```

Comment:
can this test run on vm?   yaml requires 6 servers

Reply:
Yes. All the rebuild tests are using 6 ranks, it is fine to run on vm.